tests/test_context.py

```diff
@@ -0,0 +1,90 @@
+import json
+from pathlib import Path
+
+import pytest
+from plumbum import local
+
+import copier
+
+from .helpers import build_file_tree, git_save
+
+
+def test_exclude_templating_with_operation(
+    tmp_path_factory: pytest.TempPathFactory,
+) -> None:
+    """
+    Ensure it's possible to create one-off boilerplate files that are not
+    managed during updates via `_exclude` using the `_operation` context variable.
+    """
+    src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
+
+    template = r"{%- if _operation == 'update' %}copy-only{%- endif %}"
```

Comment:
Just to keep it simple and avoid using a raw string and stripping whitespaces when not actually needed:

```suggestion
    template = "{% if _operation == 'update' %}copy-only{% endif %}"
```